src/clients/node/README.md

```diff
@@ -88,11 +52,11 @@ if (errors.length) {
 }
 ```
 
+
+
```

Comment:
As requested, leaving the comment on the diff :) Three blanks feels like a bit too too much. On one hand, it’s generated code, so who cares. On the other hand, I found myself frequently looking at the generated code anyways, so I now prefer to just generate moderately pretty code by default.

Reply:
Thank you! There were areas printing out blank lines before newlines (because not all docs have all the same sections). Fixed now.